he

```diff
@@ -0,0 +1,284 @@
+                "auth_form_error_no_auth_methods_enabled": "מצטערים, אין לנו דרך לאמת אותכם כרגע",
+                "auth_form_error_terms_not_accepted": "עליכם לקרוא ולקבל את המדיניות כדי להמשיך",
+                "invitation_description": "הוזמנתם להצטרף ל- ${business_name}. בבקשה היכנסו או הירשמו כדי לקבל את ההזמנה.",
+                "disclaimer_clickwrap_input_label": "קבלו מדיניות",
+                "disclaimer_clickwrap_error_required": "עליכם לקרוא ולקבל את המדיניות כדי להמשיך",
+                "disclaimer_dialog_heading": "קבלו כדי להמשיך",
+                "disclaimer_dialog_description": "כדי להמשיך להירשם, אנו זקוקים שתקראו ותקבלו את המדיניות שלנו",
+                "disclaimer": "על ידי המשך, אתם מסכימים למדיניות שלנו",
+                "disclaimer_terms_of_use_link": "תנאי השימוש",
+                "disclaimer_privacy_link": "מדיניות הפרטיות"
+            },
+            "sms_verify_code_page": {
+                "page_title": "אימות רב-גורמים | ${business_name}",
+                "heading": "תאמתו את מספר הטלפון",
+                "description": "הזינו את הקוד ששלחנו הרגע ל- ${phone_number]",
+                "code_input_label": "קוד",
+                "code_input_error_expired_code": "פג תוקף קוד האישור שלכם. בבקשה בקשו קוד חדש",
+                "code_input_error_incorrect_code": "נא להזין קוד אישור חוקי",
+                "code_input_error_incorrect_code_length": "קוד האישור צריך להיות 6 ספרות",
+                "code_input_error_resend_requested_too_soon": "לא ניתן לשלוח קוד נוסף עדיין. בבקשה נסו שוב מאוחר יותר",
+                "confirmation_code_input_error_unknown_error": "שגיאה באימות",
+                "continue_button": "אימות",
+                "didnt_receive_a_code_description": "לא קיבלתם קוד?",
+                "didnt_receive_a_code_link": "שלח מחדש את הקוד",
+                "choose_another_method_link": "בחר בשיטה אחרת"
+            },
+            "subscribe_form_widget": {
+                "page_title": "הירשמו כמנוי",
+                "heading": "הישארו מעודכנים",
+                "description": "שמעו מאיתנו על עדכוני מוצרים, חדשות ואירועים",
+                "name_input_label": "שם פרטי",
+                "name_input_error_required": "נא להזין שם,",
+                "name_input_error_exceeds_max_length": "השם ארוך מדי.",
+                "email_input_label": "דואר אלקטרוני",
+                "email_input_error_invalid": "הזינו דואר אלקטרונית חוקי,",
+                "email_input_error_required": "נדרש דואר אלקטרוני,",
+                "email_input_error_duplicate_email_found": "דואר אלקטרוני זה כבר רשום כמנוי,",
+                "continue_button": "הירשם",
+                "success_heading": "הצלחה! ברוכים הבאים לתוך הלופ.",
+                "credit_powered_by": "מופעל על ידי"
+            },
+            "verification_code_sms": {
+                "description": "${generated_code} הוא הקוד החד-פעמי שלך לכניסה ל- ${business_name}\n\n@ ${business_name} # ${generated_code]"
```

Comment:
@onderay this one seems to have the same format as the Spanish here - can we update? Sorry, it's just that is a specific format for OTPs

Reply:
@DaveOrDead will keep an eye out as its coming from the template as of yesterday